src/components/MessageBubble.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '../styles/theme.style.js';
+import { StyleSheet, Text, View, Image, TouchableOpacity, TextInputComponent, } from 'react-native';
+import TextInputContainer from '../containers/TextInputContainer';
+import { TextInput } from 'react-native-gesture-handler';
+
+
+class MessageBubble extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        isAuthor: true,
+        isSelectable: true,
+      };
+    }
+  
+    render() {
+      return (
+        <MessageBubbleContainer isAuthor={this.state.isAuthor} backgroundColor={this.props.labelColor} isSelectable={this.props.isSelectable}>
+           <WrittenMessage
+           placeholder='Message'
+       placeholderTextColor={"#D8D8D8"}
+       multiline={true}
+     />
+          <TextInputContainer placeholder='' isConfirmed={false}>
```

Comment:
What is the TextInputContainer doing here? Since WrittenMessage is already a TextInput, does it have a role in the component? Also, to add:
1. The editable attribute for the TextInput WrittenMessages, as we will have to toggle it later on with the edit button. You can initiliase the attribute with a state (i.e. isInEditMode) that is for now just initialised as false. 
2. A props for WrittenMessage called isAuthor initialised with this.props.value (if you are not the author, the edit feature is disabled)

Reply:
TextInputContainer has been removed.

Reply:
I already have an editable attribute in written messages and I have the isSelectable boolean. It is currently set to true which makes the component editable.